packages/utils/src/queryGenerators/createQueryString.ts

```diff
@@ -12,16 +13,20 @@ export type CheckedRule = {
 
 const DEFAULT_DEPTH = 1;
 
-const getTableByName = (tablesList: TableSchema[], tableName: string) =>
-  tablesList.find(({ name }) => tableName === name);
-
 export const createQueryString = (
   tablesList: TableSchema[],
-  tableName: string,
+  tableId: string,
   queryObjectConfig: QueryGeneratorConfig = {},
   prevKey: string = '',
 ): string => {
-  const { fields = [] } = getTableByName(tablesList, tableName) || {};
+  const table = tablesListSelectors.getTableById(tablesList, tableId);
+
+  if (!table) {
+    throw new SDKError(ERROR_CODES.TABLE_NOT_FOUND, PACKAGES.API_CLIENT, `Table with id ${tableId} not found`);
```

Comment:
@ej9x Why is this error from api-client?

Reply:
@vorobeez fixed